standard/standard-library.md

```diff
@@ -658,3 +652,393 @@ namespace System.Runtime.CompilerServices
     }
 }
 ```
+
+## C.4 Format Specifications
+
+The meaning of the formats, as used in interpolated string expressions (§interpolated-string-expressions), are defined in ISO/IEC 23271:2012. For convenience the following text is copied from the description of `System.IFormatable`.
+
+**This text is informative.**
+
+A *format* is a string that describes the appearance of an object when
+it is converted to a string. Either standard or custom formats can be used. A
+standard format takes the form *Axx*, where *A* is a single
+alphabetic character called the *format specifier*, and *xx* is an integer between zero and 99 inclusive, called the *precision specifier*. The format specifier controls the type
+of formatting applied to the value being represented as a string. The
+*precision specifier* controls the number 
+of significant digits or decimal places in the string, if applicable. [*Note:* For the list of standard format specifiers, see the
```

Comment:
This note needs to be changed to use the format we use elsewhere. (Currently the converter thinks it's a link.) I'd suggest making it a new paragraph - I'm not sure how "inline note which isn't in a list" works at the moment.